vitest.config.ts

```
@@ -8,9 +8,20 @@ export default defineConfig(({ mode }) => ({
     environment: 'node',
     watch: false,
     bail: 1,
-    testTimeout: 30_000, // Increased from 20s to handle slow edge function responses
-    hookTimeout: 15_000, // Increased from 8s to handle slow setup/teardown
-    retry: 3, // Increased retries for network flakiness
+    testTimeout: 30_000,
+    hookTimeout: 15_000,
+    retry: 2,
+    maxConcurrency: 10, // Reduced to prevent worker communication issues
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)